junit-platform-launcher/src/main/java/org/junit/platform/launcher/Launcher.java

```diff
@@ -81,8 +81,28 @@
 	 * collecting their results, and notify {@linkplain #registerTestExecutionListeners
 	 * registered listeners} about the progress and results of the execution.
 	 *
-	 * @param launcherDiscoveryRequest the launcher discovery request; never {@code null}
+	 * <p>The <em>deprecated</em> default implementation delegates to
+	 * {@link #execute(LauncherDiscoveryRequest, TestExecutionListener...)} with no
+	 * additional listeners.
+	 *
+	 * <p>The method will be removed in milestone M5.
 	 */
-	void execute(LauncherDiscoveryRequest launcherDiscoveryRequest);
```

Comment:
When deprecating something, we not only annotate it with `@Deprecated`, but we also used the `@deprecated` JavaDoc tag so that the information shows up in the published API documentation. See other usages in the framework for examples.